x/token/beginblocker.go

```diff
@@ -4,11 +4,10 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/okex/okchain/x/common/perf"
 	"github.com/okex/okchain/x/token/types"
-	//"github.com/okex/okchain/x/common/version"
 )
 
-// BeginBlocker Called every block
-func BeginBlocker(ctx sdk.Context, keeper Keeper) {
+// BeginBlocker is called when dapp handles with abci::BeginBlock
```

Comment:
BeginBlocker --> beginBlocker

Reply:
i just turned the public method "BeginBlocker" into a private method "beginBlock".